WcaOnRails/app/views/static_pages/faq.html.erb

```diff
@@ -1,5 +1,32 @@
-<% provide(:title, 'Frequently Asked Questions') %>
-
+<% provide(:title, t('faq.title')) %>
+
+<%
+faq_variables = {
+  profile_edit_path: profile_edit_path,
+  current_user_status: if current_user
+                         t('faq.already_logged_in')
+                       else
+                         t('faq.create_an_account', here: link_to(t('common.here'), new_user_registration_path))
+                       end,
+  current_user_claim: if current_user
```

Comment:
So, this is the second `if current_user`, it might be cleaner to put them together *after* the definition of `faq_variables`. Something like this:

```ruby
faq_variables = { ... }

if current_user
  faq_variables[:current_user_status] = ...
  faq_variables[:current_user_claim] = ...
else
  faq_variables[:current_user_status] = ...
  faq_variables[:current_user_claim] = ...
end
```

Your call, whichever you prefer.